tests/workflow_contracts/makefile_cargo_resolution_test.py

```
@@ -0,0 +1,53 @@
+    path_cargo = fake_bin / "cargo"
+    path_cargo.touch(mode=0o755)
+
+    environment = os.environ.copy()
+    environment.update(
+        {
+            "CARGO": cargo_override,
+            "PATH": f"{fake_bin}{os.pathsep}{environment['PATH']}",
+        }
+    )
+    result = subprocess.run(
+        ["make", "--no-print-directory", "-n", "check-fmt"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)